tools/rna_tools/targetfinder/TargetFinder.xml

```diff
@@ -0,0 +1,66 @@
+<tool id="targetfinder" name="targetfinder" version="0.1.0">
+    <requirements>
+        <requirement type="package" version="1.07">targetfinder</requirement>
+        <requirement type="package" version="35.4.12">fasta</requirement>
+    </requirements>
+    <stdio>
+        <exit_code range="1:" />
+    </stdio>
+
+    <command><![CDATA[
+        targetfinder.pl
+            -s "$s"
+            -d $d
+            -q "$q"
+            -c $c
+            -t $t
+            -p "$p"
+            $r
+            > $output1
+    ]]></command>
+    <inputs>
+        <param label="Small RNA sequence (RNA or DNA, 5'->3')" help="(-s)" name="s" type="text"/>
+        <param label="Target sequence database file (FASTA-format)" help="(-d)" name="d" type="data" format="fasta" />
+        <param label="Query sequence name (DEFAULT = 'query')" help="(-q)" name="q" type="text" value="query" />
+        <param label="Prediction score cutoff value (DEFAULT = 4)" help="(-c)" name="c" type="float" value="4.0" />
+        <param label="Threads for parallel Smith-Waterman searches (DEFAULT = 1)" help="(-t)" name="t" type="integer" value="1" />
+        <param label="Output format for small RNA-target pairs." help="(-p)" name="p" type="select" >
+                <option value="classic">classic</option>
+                <option value="gff">gff</option>
+                <option value="json">json</option>
+                <option value="table">table</option>
```

Comment:
:) Just saying that we should help the user here. Describe the output format a little bit, what a user can expect.
Or mention this in the help.